src/ui/screens/Bounty/BountiesScreen.tsx

```diff
@@ -20,12 +20,16 @@ const ItemSeparator = () => <Padder scale={0.5} />;
 
 export function BountiesScreen({navigation}: ScreenProps) {
   const {openBottomSheet, closeBottomSheet, BottomSheet} = useBottomSheet();
-  const {data: bounties, loading} = useBounties();
+  const {data: bounties, loading, refetch: refetchBounties} = useBounties();
 
   const toBountyDetails = (index: string) => {
     navigation.navigate(bountyDetailScreen, {index});
   };
 
+  const bountiesRefetch = () => {
```

Comment:
Why this extra function?

Reply:
If the issue was a typescript error, then lets put the function back to avoid coupling the type with Apollo, that's too specific for a generic prop

Reply:
I have updated the code now, it decoupled and the type is added to the hook.